Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
     name: "swift-algorand-sdk",
     platforms: [
           .macOS(.v10_12),
-           .iOS(.v10),
+           .iOS(.v14),
```

Comment:
why iOS 14? This may be not acceptable for many use cases. Is there any reason for this?